api/src/apps/ovfiets/importer.py

```diff
@@ -0,0 +1,145 @@
+            key = self.altered_keys[key]
+        return key
+
+    def set_to_model_field(self, model_instance, key, value):
+        '''
+        Checks if field exists then saves it to the model instance
+
+        Specific to this datasource, we dump all non existing fields to
+        the jsonfield `unmapped`
+        '''
+        try:
+            field = model_instance._meta.get_field(key)
+            setattr(model_instance, field.name, value)
+        except FieldDoesNotExist:
+            log.debug(f"Field {key} does not exist. Adding to unmapped")
+            model_instance.unmapped.update({key: value})
+
+    def clean_value(self, key, value):
+        '''
+        Parses the fetch_time timestamp and makes aware (timezone)
+        '''
+        if key == 'fetch_time':
+            value = timezone.datetime.fromtimestamp(value, tz=timezone.utc)
+        return key, value
+
+    def extract_extra(self, data_instance):
+        '''
+        Extract the nested pairs since it contains
+        the bikeRentals and fetch_time which we save
+        in the model.
+        '''
+        extra = data_instance.pop('extra')
+        data_instance.update(extra)
+        return data_instance
+
+    def extract_coordinates(self, data_instance):
+        '''
+        Extract the coordinates values and create
+        a Point object
+        '''
+        data_instance.update({
+            'geometrie': Point(
+                data_instance.pop('lng'),
+                data_instance.pop('lat')
+            )
+        })
+        return data_instance
+
+    def build_model_instance(self, data_instance, timestamp):
+        '''
+        Build a model instance with the data instance items
+        after cleaning the items.
+        '''
+        model_instance = OvFiets(scraped_at=timestamp, unmapped={})
+        data_instance = self.extract_extra(data_instance)
+        data_instance = self.extract_coordinates(data_instance)
+
+        for key, value in data_instance.items():
+            key = self.clean_key(key)
+            key, value = self.clean_value(key, value)
+            self.set_to_model_field(model_instance, key, value)
+
+        model_instance.stadsdeel = District.objects.get_district(model_instance.geometrie)
+
+        # If no district then it is not in Amsterdam. Skipping the neighbourhood saves computing time.
+        if model_instance.stadsdeel:
+            model_instance.buurt_code = Neighbourhood.objects.get_neighbourhood(model_instance.geometrie)
+
+        return model_instance
+
+    def queryset_to_model_instance_list(self, queryset):
+        '''
+        Creates a model instance list from the queryset.
+
+        Process:
+            - Create empty model_instance list
+            - Loop through the raw_instances in the queryset
+            - Loop through the data_instances in the raw_instance json field
+            - Build a model_instance
+            - Append to lis
+            - Repeat from step 2
```

Comment:
You might want to use a numbered list if you are going to refer to it by number. (e.g. 1, 2,, 3 instead of dashes)